app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -91,8 +94,10 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
   router.get('/select-solution/select-price/select-recipient', authProvider.authorise({ claim: 'ordering' }), withCatch(authProvider, async (req, res) => {
     const { orderId } = req.params;
 
+    const context = await getSolutionRecipientPageContext({ orderId, solutionName: 'Solution One' });
```

Comment:
where should this solution name be coming from?

Reply:
I'm going to do this part of the next PR but it will be coming from session. When a solution is selected we'll also need to store the solutionName too and retrieve that here 😢